static/css/bureau.less

```diff
@@ -253,21 +266,17 @@
 }
 
 .youtube-player_image {
```

Comment:
Can we group the positioning and display properties separately? It makes it easier to find where different things are later, especially in something like this with a lot of css properties.

https://github.com/cfpb/front-end/blob/master/css.md#property-order